prometheusbp/metrics.go

```diff
@@ -0,0 +1,7 @@
+package prometheusbp
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var DefaultBuckets = prometheus.ExponentialBuckets(0.0001, 1.5, 26) // 100us ~ 2.5s
```

Comment:
@bjk-reddit do you think these bucket values are ok?

Reply:
I'm wondering if this is a bit too granular to have as a default. Looking at one of our first services, this generates 19,000 metrics due to the number of methods/pods/etc.

What we we start out with something reduced to 16 buckets, with a range of 100us to ~15s with this:

```suggestion
var DefaultBuckets = prometheus.ExponentialBuckets(0.0001, 2.5, 14) // 100us ~ 14.9s
```

This will give us reasonable granularity from very fast responses to our front-end timeout. This will be more useful for services that depends on slow backends.

Reply:
🔕 @bjk-reddit: will 100us be small enough? it probably is for rpc calls, but we might be using some local spans in the future and those can be much faster. not sure if we want the same default buckets to work for all.

Reply:
@fishy I think 100us is probably fine for a default lower bound. We're mostly concerned about things being slower, so counting things in buckets when it's longer is the more important part.  If you think about it from an SLO perspective (which is what the metrics are about), knowing the difference between 50us and 100us is "more than fast enough" from a full RPC reply.

Here's an example of the data from the new preference service.

<img width="640" alt="Screen Shot 2021-11-03 at 17 19 38" src="https://user-images.githubusercontent.com/86239705/140100312-054395da-e02b-48a5-a7d6-a641229e480e.png">
 
There are a few requests under 100us, but only sub 10 RPS compared to the 100,000 RPS in the 300us-2ms range.

Reply:
I'm mainly talking about some of the local spans that the majority (or all) of the data points are below 100us. Those usually don't have an SLA/SLO, but developers still care about latency data.

Reply:
Those are probably not something we would include in metrics, only tracing spans. The main goal of metrics and these buckets are for real-time monitoring for our SLO/SLI use cases.